src/cljs/aikakonematka/game.cljs

```diff
@@ -194,6 +197,9 @@
                   (sound/play-beep! (sound/frequencies-of-major-scale-in-4th-octave
                                       (mod (+ 1 util/row-col-num col)
                                            (count sound/frequencies-of-major-scale-in-4th-octave))))
+                  (send-music-note-fn! (sound/frequencies-of-major-scale-in-4th-octave
```

Comment:
(sound/frequencies-of-major-scale-in-4th-octave
+                                         (mod (+ 1 util/row-col-num col)
+                                              (count sound/frequencies-of-major-scale-in-4th-octave)))
can be reused with local binding